semantic_conventions/trace/instrumentation/aws-sdk.yml

```
@@ -0,0 +1,1195 @@
+groups:
+  - id: awssdk
+    prefix: awssdk
+    brief: >
+      These conventions apply to operations using the AWS SDK. They map request or response parameters
+      in AWS SDK API calls to attributes on a Span. The conventions have been collected over time based
+      on feedback from AWS users of tracing and will continue to increase as new interesting conventions
+      are found.
+
+      Some descriptions are also provided for populating OpenTelemetry semantic conventions.
+    attributes:
+      - id: service
+        type: string
+        brief: The service name of the request, as returned by the AWS SDK
+        examples:
+          - DynamoDB
+          - S3
+      - id: operation
+        type: string
+        brief: The operation name of the request, as returned by the AWS SDK
+        examples:
+          - GetItem
+          - PutItem
+
+  - id: dynamodb.batchgetitem
+    brief: DynamoDB.BatchGetItem
+    extends: awssdk
+    attributes:
+      - id: table_names
+        type: string
+        brief: Extract the keys of the `RequestItems` object field in the request
```

This is worded like an instruction for instrumentation writers. Since backend writers will also be interested in what this is, I suggest

```suggestion
        brief: "The keys in the request's `RequestItems` object field."
```

---

I'm also not sure what this means (is it an object or string field now?) but I assume someone familiar with AWS would know.